### ISSUE TITLE
Content label may not be readable by screen readers

### ISSUE THREAD

**Issue by Abbybretado** [AmericaSCORESBayArea/AmericaScores-CoachApp#481]
Login Error Message may not be readable by screen readers

1. Go to Open America Scores Attendance App
2. Try signing in with non-existent or incorrect account information.
3.. Login Error dialog window appears and may not be readable by screen readers

Expected behavior
Login Error dialog window content should be read by screen readers.

Screenshots
<img width="226" alt="image" src="https://github.com/AmericaSCORESBayArea/AmericaScores-CoachApp/assets/165066898/66190fe4-b320-46f8-8281-287877223ef1">

Smartphone (please complete the following information):
 - Device: Pixel 6
 - OS: Android 12 (SDK 31)
 - Browser n/a
 - App Version 
- Public or Test version: From Google Play Console review
